image/README.md

```diff
@@ -0,0 +1,119 @@
+# @magenta/image
+
+This JavaScript implementation of Magenta's image models uses [TensorFlow.js](https://js.tensorflow.org) for GPU-accelerated inference.
+
+Complete documentation is available at https://tensorflow.github.io/magenta-js/image.
+
+## Contents
+
+- [Example Applications](#example-applications)
+- [Supported Models](#supported-models)
+- [Getting Started](#getting-started)
+
+## Example Applications
+
+Here are a few applications built with `@magenta/image`:
```

Comment:
you can remove this section until you have a demo to link to, wdyt?

Reply:
Done.